From 0a31ca263b90bfbffb8b3adbc14fe41577a3f9dd Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Mon, 18 Jan 2021 15:48:42 -0600
Subject: [PATCH] remove -ffast-math

---
 linux/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/linux/Makefile b/linux/Makefile
index fff4031..dea46d6 100644
--- a/linux/Makefile
+++ b/linux/Makefile
@@ -75,12 +75,12 @@ CC=gcc
 BASE_CFLAGS=-I/usr/local/include -Dstricmp=strcasecmp -D_GNU_SOURCE -Wno-format-truncation -Wno-unused-result
 
 ifeq ($(ARCH),axp)
-RELEASE_CFLAGS=$(BASE_CFLAGS) -DNDEBUG -O3 -ffast-math -funroll-loops \
+RELEASE_CFLAGS=$(BASE_CFLAGS) -DNDEBUG -O3 -funroll-loops \
 	-fomit-frame-pointer -fexpensive-optimizations
 else ifeq ($(ARCH),e2k)
-RELEASE_CFLAGS=$(BASE_CFLAGS) -DNDEBUG -O3 -ffast-math -ffast
+RELEASE_CFLAGS=$(BASE_CFLAGS) -DNDEBUG -O3 -ffast
 else
-RELEASE_CFLAGS=$(BASE_CFLAGS) -DNDEBUG -O3 -ffast-math -funroll-loops \
+RELEASE_CFLAGS=$(BASE_CFLAGS) -DNDEBUG -O3 -funroll-loops \
 	-fomit-frame-pointer -fexpensive-optimizations
 endif
 
-- 
2.29.2

