From 21ec466485b25d3cb89ea74073fc56ec8623bde6 Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Mon, 24 Apr 2023 15:37:09 -0500
Subject: [PATCH] fluidsynth - allow pulseaudio and others

---
 ports/fluidsynth/portfile.cmake | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/ports/fluidsynth/portfile.cmake b/ports/fluidsynth/portfile.cmake
index 3a8c18567..b3c6e0574 100644
--- a/ports/fluidsynth/portfile.cmake
+++ b/ports/fluidsynth/portfile.cmake
@@ -19,11 +19,11 @@ vcpkg_check_features(
 # and disable all other features to avoid system libraries to be picked up
 set(WINDOWS_OPTIONS enable-dsound enable-wasapi enable-waveout enable-winmidi HAVE_MMSYSTEM_H HAVE_DSOUND_H HAVE_OBJBASE_H)
 set(MACOS_OPTIONS enable-coreaudio enable-coremidi COREAUDIO_FOUND COREMIDI_FOUND)
-set(LINUX_OPTIONS enable-alsa ALSA_FOUND)
+set(LINUX_OPTIONS enable-alsa enable-pulseaudio enable-pipewire enableALSA_FOUND)
 set(ANDROID_OPTIONS enable-opensles OpenSLES_FOUND)
 set(IGNORED_OPTIONS enable-coverage enable-dbus enable-floats enable-fpe-check enable-framework enable-jack enable-lash
-    enable-libinstpatch enable-midishare enable-oboe enable-openmp enable-oss enable-pipewire enable-portaudio
-    enable-profiling enable-pulseaudio enable-readline enable-sdl2 enable-systemd enable-trap-on-fpe enable-ubsan)
+    enable-libinstpatch enable-midishare enable-oboe enable-openmp enable-oss enable-portaudio
+    enable-profiling enable-readline enable-systemd enable-trap-on-fpe enable-ubsan)
 
 if(VCPKG_TARGET_IS_WINDOWS)
     set(OPTIONS_TO_ENABLE ${WINDOWS_OPTIONS})
-- 
2.40.0

