From ae6b8f53874a08bb485d1da2cf97aa85c506ee3e Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Fri, 1 Jul 2022 10:55:19 -0500
Subject: [PATCH] Compile fix

---
 source/build/src/polymer.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/source/build/src/polymer.cpp b/source/build/src/polymer.cpp
index d98c1fc27..36fb1ce95 100644
--- a/source/build/src/polymer.cpp
+++ b/source/build/src/polymer.cpp
@@ -5336,9 +5336,9 @@ static int32_t      polymer_bindmaterial(const _prmaterial *material, const int1
             // farther far clipoff distance to account for the fact that the
             // distance to the fragment is the common Euclidean one, as opposed to
             // the "ortho" distance of Build.
-            static constexpr float factor_new = 1.f / ((2048.f * (1.07f / 1.024f) * (150.f / 230.f) / 35.f) * material_visibility_divisor);
+            static float factor_new = 1.f / ((2048.f * (1.07f / 1.024f) * (150.f / 230.f) / 35.f) * material_visibility_divisor);
 
-            static constexpr float factor_old = 1.f / ((2048.f * (1.07f / 1.024f) / 35.f) * material_visibility_divisor);
+            static float factor_old = 1.f / ((2048.f * (1.07f / 1.024f) / 35.f) * material_visibility_divisor);
 
             glUniform1f(prprogram.uniform_visibility, globalvisibility * material->visibility * r_usenewshading > 1 ? factor_new : factor_old);
         }
-- 
2.36.1

