From b5fe701c3243ee8520f98a5c5e4eaf4fe90e5a01 Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Wed, 29 Mar 2023 22:03:25 -0500
Subject: [PATCH 1/1] allow unlimited saves

---
 src/avp/savegame.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/avp/savegame.c b/src/avp/savegame.c
index 5125995..a70f9d9 100644
--- a/src/avp/savegame.c
+++ b/src/avp/savegame.c
@@ -380,6 +380,8 @@ static BOOL SaveGameAllowed()
 		default: ;
 	}
 
+	return TRUE;
+
 
 	if(!NumberOfSavesLeft)
 	{
-- 
2.40.0

