From 1324fddccb005ca081bf70bd79966fd3c066d887 Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Tue, 4 Aug 2020 22:13:05 -0500
Subject: [PATCH 1/1] Add ultrawide

---
 src/main.c  | 3 ++-
 src/main2.c | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/main.c b/src/main.c
index 4ca3465..8df998f 100644
--- a/src/main.c
+++ b/src/main.c
@@ -256,7 +256,8 @@ VideoModeStruct VideoModeList[] = {
 {	1280,	1024,	0	},
 {	1600,	1200,	0	},
 {   1920,   1080,   0   },
-{   1920,   1200,   0   }
+{   1920,   1200,   0   },
+{   2560,   1080,   0   }
 };
 
 int CurrentVideoMode;
diff --git a/src/main2.c b/src/main2.c
index 46efa51..1a7ffd4 100644
--- a/src/main2.c
+++ b/src/main2.c
@@ -327,7 +327,8 @@ VideoModeStruct VideoModeList[] = {
 	{	1280,	960,	0	},
 	{	1280,	1024,	0	},
 	{	1600,	1200,	0	},
-	{	1920,	1080,	0	}
+	{	1920,	1080,	0	},
+    {	2560,	1080,	0	}
 };
 
 int CurrentVideoMode;
-- 
2.27.0

