From 92556965fa640d8cb1cb0b00ecc00a57e1c3e93e Mon Sep 17 00:00:00 2001
From: Patryk Obara <dreamer.tan@gmail.com>
Date: Tue, 10 Sep 2019 12:53:16 +0200
Subject: [PATCH] Workaround for missing PRId64

---
 src/d_net.cpp | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/src/d_net.cpp b/src/d_net.cpp
index 95e1cd756..cd43ecffb 100644
--- a/src/d_net.cpp
+++ b/src/d_net.cpp
@@ -2928,6 +2928,11 @@ int Net_GetLatency(int *ld, int *ad)
 	return severity;
 }
 
+#ifndef PRId64
+#  define LUX_PRID_WORKAROUND
+#  define PRId64 "lld"
+#endif
+
 // [RH] List "ping" times
 CCMD (pings)
 {
@@ -2938,6 +2943,11 @@ CCMD (pings)
 					players[i].userinfo.GetName());
 }
 
+#ifdef LUX_PRID_WORKAROUND
+#  undef PRId64
+#  undef LUX_PRID_WORKAROUND
+#endif
+
 //==========================================================================
 //
 // Network_Controller
-- 
2.27.0

