From 0dffa6f48ef12152b4b92fdb976e555b7818c7ef Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Wed, 1 Jul 2020 23:38:12 -0500
Subject: [PATCH] Make windowed the default for first launch

---
 src/core/Config.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/Config.cpp b/src/core/Config.cpp
index 01b1509a2..560402461 100644
--- a/src/core/Config.cpp
+++ b/src/core/Config.cpp
@@ -81,7 +81,7 @@ const int
 	quickLevelTransition = JumpToChangeLevel;
 
 const bool
-	fullscreen = true,
+	fullscreen = false,
 	viewBobbing = true,
 	screenShake = true,
 	showCrosshair = true,
-- 
2.27.0

