From 0a45e98127af247abf1720965b88766a58bd0dbb Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Mon, 4 Jul 2022 21:14:16 -0500
Subject: [PATCH] Fix compile error

---
 src/SDL/rinit.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/SDL/rinit.c b/src/SDL/rinit.c
index d3bdaca..7e588d5 100644
--- a/src/SDL/rinit.c
+++ b/src/SDL/rinit.c
@@ -19,7 +19,6 @@
 #include "rinit.h"
 #include "devstats.h"
 #include "Types.h"
-#include "debug.h"
 
 
 extern unsigned int mainSoftwareDirectDraw;
-- 
2.37.0

