From 6a54078c793b6254dedc1600d98a2a4ce4796ae5 Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Sat, 19 Mar 2022 12:52:24 -0500
Subject: [PATCH 1/1] Compile error fix

---
 disabled.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/disabled.h b/disabled.h
index 677d52a..f85aefe 100644
--- a/disabled.h
+++ b/disabled.h
@@ -17,11 +17,7 @@ inline bool disabledScan(QString str, int val) {
         // 0x050C08 is the Qt version number (5.12.8) in hexadecimal. This is
         // the Qt version available in Kubuntu 20.04. It comes from
         // qtcoreversion.h
-#if QTCORE_VERSION <= 0x050C08
         for (auto s : str.split(",", QString::SkipEmptyParts))
-#else
-        for (auto s : str.split(",", Qt::SkipEmptyParts))
-#endif
 	{
                 bool ok;
 
-- 
2.35.1

