From 93154e966ebf3b660baefcd7adf4095d687f97e5 Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Sat, 15 Apr 2023 23:00:35 -0500
Subject: [PATCH 1/1] compile fix

---
 src/core/system.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/core/system.h b/src/core/system.h
index 3c83b9c5..bef92ad1 100644
--- a/src/core/system.h
+++ b/src/core/system.h
@@ -5,7 +5,8 @@
 #ifdef	__cplusplus
 extern "C" {
 #endif
-    
+
+#include <unistd.h>
 #include <stdint.h>
 #include <SDL2/SDL_platform.h>
 #include <SDL2/SDL_opengl.h>
-- 
2.40.0

