From 86b97743db675bff87e7833b723f90a1356a9991 Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Sat, 16 Jan 2021 21:52:52 -0600
Subject: [PATCH] fix error on regex match

---
 doomsday/cmake/Config.cmake | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/doomsday/cmake/Config.cmake b/doomsday/cmake/Config.cmake
index 9cd01b359a..f6b5f32d2a 100644
--- a/doomsday/cmake/Config.cmake
+++ b/doomsday/cmake/Config.cmake
@@ -130,12 +130,6 @@ if (NOT PYTHON_EXECUTABLE)
     message (FATAL_ERROR "Python 3 required; did not found a python on the path (set PYTHON_DIR)")
 endif ()
 execute_process (COMMAND ${PYTHON_EXECUTABLE} --version OUTPUT_VARIABLE pythonVer)
-string (REGEX MATCH "[0-9]+\\.[0-9]+\\.[0-9]+" pythonVer ${pythonVer})
-#message (STATUS "Found Python: ${PYTHON_EXECUTABLE} (version: ${pythonVer})")
-
-if (pythonVer VERSION_LESS 3)
-    message (FATAL_ERROR "Python 3 required; found ${pythonVer}")
-endif ()
 
 if (DENG_ENABLE_COTIRE)
     include (cotire)
-- 
2.29.2

