From 64a55fe395c7825ce39e5c35d405fe86055c25bc Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Fri, 10 Sep 2021 22:17:26 -0500
Subject: [PATCH] Remove nuklear menu

---
 vid_sdl.c | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/vid_sdl.c b/vid_sdl.c
index 9edc8aa..9766e82 100644
--- a/vid_sdl.c
+++ b/vid_sdl.c
@@ -22,8 +22,8 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 #include "SDL/SDL_opengl.h"
 
 //#include "loader.h"
-//#include "SDL/SDL.h"
-//#include "SDL/SDL_opengl.h"
+#include "SDL/SDL.h"
+#include "SDL/SDL_opengl.h"
 //#define NK_INCLUDE_FIXED_TYPES
 //#define NK_INCLUDE_STANDARD_IO
 //#define NK_INCLUDE_STANDARD_VARARGS
@@ -35,7 +35,6 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 //#define NK_SDL_GL2_IMPLEMENTATION
 //#include "nuklear/nuklear.h"
 //#include "nuklear/nuklear_sdl_gl2.h"
-#include "mymenu.h"
 //#include "curl/curl.h"
 
 #include <stdio.h>
@@ -1660,9 +1659,6 @@ static qboolean VID_InitModeGL(viddef_mode_t *mode)
 
 	GL_Clear(GL_COLOR_BUFFER_BIT, NULL, 1.0f, 0);	// clear to black (loading plaque will be seen over this)
 
-	//menu
-	InitMenu(window);
-
 
 	SDL_SetWindowPosition(window, xPos, yPos);
 	SDL_SetWindowSize(window, mode->width, mode->height);
-- 
2.33.0

