From 5eacb388e7eb3b20f03c776f9dc19accb3df03e9 Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Sat, 8 Aug 2020 10:47:21 -0500
Subject: [PATCH 1/1] borderless full screen

---
 src/xrEngine/Device_create.cpp | 19 ++-----------------
 1 file changed, 2 insertions(+), 17 deletions(-)

diff --git a/src/xrEngine/Device_create.cpp b/src/xrEngine/Device_create.cpp
index 3bbf172fa..ffe778329 100644
--- a/src/xrEngine/Device_create.cpp
+++ b/src/xrEngine/Device_create.cpp
@@ -117,25 +117,10 @@ void CRenderDevice::UpdateWindowProps(const bool windowed)
     }
     else
     {
-        // Changing monitor, unset fullscreen for the previous monitor
-        if (SDL_GetWindowDisplayIndex(m_sdlWnd) != Vid_SelectedMonitor)
-            SDL_SetWindowFullscreen(m_sdlWnd, SDL_DISABLE);
-
-        SDL_Rect rect;
-        SDL_GetDisplayBounds(Vid_SelectedMonitor, &rect);
-        SDL_SetWindowSize(m_sdlWnd, psCurrentVidMode[0], psCurrentVidMode[1]);
-        SDL_SetWindowPosition(m_sdlWnd, rect.x, rect.y);
-
         if (b_is_Ready)
         {
-            SDL_SetWindowFullscreen(m_sdlWnd, SDL_WINDOW_FULLSCREEN);
-
-            SDL_DisplayMode mode;
-            SDL_GetWindowDisplayMode(m_sdlWnd, &mode);
-            mode.w = psCurrentVidMode[0];
-            mode.h = psCurrentVidMode[1];
-            mode.refresh_rate = Vid_SelectedRefreshRate;
-            SDL_SetWindowDisplayMode(m_sdlWnd, &mode);
+            SDL_SetWindowPosition(m_sdlWnd, 0, 0);
+            SDL_SetWindowFullscreen(m_sdlWnd, SDL_WINDOW_FULLSCREEN_DESKTOP);
         }
     }
 
-- 
2.27.0

