From 2856861a10ebd11c45d33e73a0dbff3c74c4ba1f Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Fri, 10 Jul 2020 09:48:44 -0500
Subject: [PATCH] Fix crash on exit of game

---
 src/xrGame/GamePersistent.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/xrGame/GamePersistent.cpp b/src/xrGame/GamePersistent.cpp
index ce8f44280..377993545 100644
--- a/src/xrGame/GamePersistent.cpp
+++ b/src/xrGame/GamePersistent.cpp
@@ -274,7 +274,9 @@ void CGamePersistent::OnAppEnd()
 
     GMLib.Unload();
 
+#ifdef XR_PLATFORM_WINDOWS
     xr_delete(ansel);
+#endif
 }
 
 void CGamePersistent::Start(LPCSTR op) { inherited::Start(op); }
-- 
2.27.0

