From ecf822929bff22f843ab75ef0240a3892a398cab Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Mon, 13 Jul 2020 17:31:24 -0500
Subject: [PATCH] Force 32-bit

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index e9b2930..31c7014 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
-CFLAGS = -fPIC -Wall -Wextra -O2 `sdl2-config --cflags` -fvisibility=hidden -g
-LDFLAGS = -shared -ldl
+CFLAGS = -fPIC -Wall -Wextra -O2 -m32 `sdl2-config --cflags` -fvisibility=hidden -g
+LDFLAGS = -shared -ldl -m32
 TARGET = libSDL-1.2.so.0
 
 SRCS = main.c video.c yuv.c cursor.c audio.c audiocvt.c timer.c events.c \
-- 
2.27.0

