From 172f90a39c05e4537bde6dbe96de23808975c0a5 Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Thu, 14 Oct 2021 14:02:39 -0500
Subject: [PATCH 1/1] Fix compile error

---
 components/detournavigator/offmeshconnectionsmanager.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/components/detournavigator/offmeshconnectionsmanager.hpp b/components/detournavigator/offmeshconnectionsmanager.hpp
index 20a6427cd..2b2e64b4f 100644
--- a/components/detournavigator/offmeshconnectionsmanager.hpp
+++ b/components/detournavigator/offmeshconnectionsmanager.hpp
@@ -8,6 +8,7 @@
 
 #include <components/misc/guarded.hpp>
 
+#include <algorithm>
 #include <map>
 #include <unordered_set>
 #include <vector>
-- 
2.33.0

