From 4e3c0702039d57114dcf499dcae9b05e1750a978 Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Thu, 2 Jun 2022 23:46:48 -0500
Subject: [PATCH] force local path

---
 components/files/linuxpath.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/components/files/linuxpath.cpp b/components/files/linuxpath.cpp
index c3dead296..7967ccc92 100644
--- a/components/files/linuxpath.cpp
+++ b/components/files/linuxpath.cpp
@@ -73,8 +73,8 @@ boost::filesystem::path LinuxPath::getCachePath() const
 
 boost::filesystem::path LinuxPath::getGlobalConfigPath() const
 {
-    boost::filesystem::path globalPath(GLOBAL_CONFIG_PATH);
-    return globalPath / mName;
+    boost::filesystem::path localPath = getLocalPath();
+    return localPath;
 }
 
 boost::filesystem::path LinuxPath::getLocalPath() const
@@ -97,8 +97,8 @@ boost::filesystem::path LinuxPath::getLocalPath() const
 
 boost::filesystem::path LinuxPath::getGlobalDataPath() const
 {
-    boost::filesystem::path globalDataPath(GLOBAL_DATA_PATH);
-    return globalDataPath / mName;
+    boost::filesystem::path localPath = getLocalPath();
+    return localPath;
 }
 
 boost::filesystem::path LinuxPath::getInstallPath() const
-- 
2.36.0

