From 09b4c25230e5993435a193c78fab55a978a3f57f Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Mon, 27 Sep 2021 15:13:31 -0500
Subject: [PATCH 1/1] Workaround for force link of lgcc

---
 configure.ac | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 8a691b64..e1d5b181 100644
--- a/configure.ac
+++ b/configure.ac
@@ -343,10 +343,6 @@ else
     LDFLAGS_NOSTARTFILES="-XCClinker -nostartfiles"
 fi
 
-if test x$GCC = xyes -a x$intel_compiler != xyes -a x$qcc_compiler != xyes -a x$android != xyes; then
-  LIBCRTS="-lgcc_s"
-fi
-
 AC_MSG_CHECKING([for __builtin___clear_cache])
 AC_LINK_IFELSE(
   [AC_LANG_PROGRAM([[]], [[__builtin___clear_cache(0, 0)]])],
-- 
2.33.0

