From d0beb717cc95008815eb7bdeb2459c4d25f75e59 Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Fri, 10 Jul 2020 17:06:11 -0500
Subject: [PATCH] Workaround for missing PRId64

---
 framework/fs/physfs_archiver_cue.cpp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/framework/fs/physfs_archiver_cue.cpp b/framework/fs/physfs_archiver_cue.cpp
index f0211fef..745fabdd 100644
--- a/framework/fs/physfs_archiver_cue.cpp
+++ b/framework/fs/physfs_archiver_cue.cpp
@@ -20,6 +20,11 @@
 #include <map>
 #include <physfs.h>
 
+#ifndef PRId64
+#  define LUX_PRID_WORKAROUND
+#  define PRId64 "lld"
+#endif
+
 using namespace OpenApoc;
 
 namespace
-- 
2.27.0

