From 77bc69ea653fdf6dd035888ade05733dd84deda2 Mon Sep 17 00:00:00 2001
From: d10sfan <d10sfan@gmail.com>
Date: Fri, 10 Jul 2020 17:12:10 -0500
Subject: [PATCH] Workaround for missing PRIu64

---
 framework/fs/physfs_archiver_cue.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/framework/fs/physfs_archiver_cue.cpp b/framework/fs/physfs_archiver_cue.cpp
index 745fabdd..8e8a0347 100644
--- a/framework/fs/physfs_archiver_cue.cpp
+++ b/framework/fs/physfs_archiver_cue.cpp
@@ -22,6 +22,7 @@
 
 #ifndef PRId64
 #  define LUX_PRID_WORKAROUND
+#  define PRIu64 "llu"
 #  define PRId64 "lld"
 #endif
 
-- 
2.27.0

